release-notes.txt

```diff
@@ -0,0 +1,237 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-15
+
+! New in this release
+- baseboxd:
+  - updated version to 1.11.1
+  - added support for Spanning Tree Protocol (STP, RSTP and MSTP) on bonded interfaces.
+  - added initial support for Broadcom KNET interfaces.
+- Linux kernel updated to version 5.15.22
+- iproute2: udpate to version 5.16
+
+! Experimental features
+- OF-DPA: initial support for creating and enabling Broadcom KNET interfaces.
+
+! Bug fixes
```

Comment:
i think linking to the issues created (and now resolved) on github would be helpful here

Reply:
Linked to github issues/mr where it made sense

Reply:
lgtm